tst/suites/gpu.suite

```diff
@@ -19,4 +19,4 @@ binary/binary
 binary_adi/binary_adi
 nbody/nbody
 diffusion/viscous_diffusion
-diffusion/thermal_diffusion
+#diffusion/thermal_diffusion
```

Comment:
Time constraint?

Reply:
Yeah this takes like 2000 seconds, because its many cycles and few cells